src/screens/GreenBlock/GreenBlock.tsx

```diff
@@ -27,6 +27,18 @@ function GreenBlock({navigation, route}: Props) {
   const greenBlockIdToJoin = route.params?.greenBlockIdToJoin;
   const alertPending = useRef(false);
 
+  const shouldNavigateToTree = useCallback(() => {
+    if (route.params && route.params.goTree) {
```

Comment:
You can use optional chaining like so `route.params?.goTree`

Reply:
I got it.